.github/workflows/build-yocto.yml

```diff
@@ -21,16 +21,11 @@ jobs:
     - name: Build the Docker image
       run: cd Build && ./build.sh; 
     
-    - name: Run the Docker image and build output files with SDK
```

Comment:
Did we want to build the SDK as part of a release too though?

Reply:
I don't think so